### ISSUE TITLE
Can not reproduce regular time series datasets results

### ISSUE THREAD

**Issue by RaptorMai** [Leezekun/ViTST#12]
Thanks for releasing the code. I tried to reproduce the regular time series dataset results but I found that I could not get similar number as the reported ones in the paper. I wonder if you could share the script to reproduce the regular data results. 

For example, for EthanolConcentration, this is the script I ran:

`python3 run_ImgCLS.py \
    --model swin\
    --seed 1799 \
    --save_total_limit 1 \
    --dataset EthanolConcentration \
    --dataset_prefix differ_-*1_**1_2*2_256*256_images \
    --train_batch_size 48 \
    --eval_batch_size 128 \
    --logging_steps 20 \
    --save_steps 20 \
    --epochs 20 \
    --learning_rate 1e-4 \
    --n_runs 1 \
    --n_splits 5 \
    --do_train`

I only got Accuracy  = 33.0 +/- 0.8, which is quite different from the reported 45.6.
